src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -307,15 +307,15 @@ You can create, read, update, or delete obfuscation expressions by using the New
        <img
          title="Create an obfuscation expression using the Logs UI"
          alt="Create an obfuscation expression using the Logs UI"
-         src={obfuscationCreateExpressionUi}
+         src={ObfuscationCreateExpression}
        />
 
        **Using NerdGraph:**
 
        Use the `logConfigurationsCreateObfuscationExpression` mutator under `logConfigurations`. Refer to the populated example in GraphiQL as well as the related documentation [in **api.newrelic.com/graphiql**](https://api.newrelic.com/graphiql#query=mutation%20%7B%0A%20%20logConfigurationsCreateObfuscationExpression%28accountId%3A%201234%2C%20expression%3A%20%7Bname%3A%20%22Credit%20Card%20Number%22%2C%20regex%3A%20%22%28%5C%5Cd%7B4%7D-%5C%5Cd%7B4%7D-%5C%5Cd%7B4%7D-%5C%5Cd%7B4%7D%29%22%2C%20description%3A%20%22Reusable%20expression%20to%20capture%20credit%20card%20numbers%22%7D%29%20%7B%0A%20%20%20%20id%0A%20%20%20%20name%0A%20%20%20%20regex%0A%20%20%20%20description%0A%20%20%20%20createdAt%0A%20%20%20%20createdBy%20%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%20%20email%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A).
 
        <Callout variant="important">
-         You must escape the `regex` when introducing it via NerdGraph: `(\\d{4}-\\d{4}-\\d{4}-\\d{4})`
+         You must escape the `regex` when introducing it via NerdGraph: `((?:(?:4\\d{3})|(?:5[1-5]\\d{2})|6(?:011|5[0-9]{2}))(?:-?|\\040?)(?:\\d{4}(?:-?|\\040?)){3}|(?:3[4,7]\\d{2})(?:-?|\\040?)\\d{6}(?:-?|\\040?)\\d{5})`
```

Comment:
i'd recommend putting this in a codeblock instead of inline code

Reply:
Done and just pushed it. Let me know if that helps.